docs/getting-started/proofs-and-verification/curl/state.md

```diff
@@ -5,10 +5,10 @@ title: Proof of Existence of a State/Data
 
 ### State proof query
 
-As it mentioned in Orion description [here], Orion maintains separated persisted graph data structure for historical data transitions, so a user can execute query on those historical changes to understand the lineage of each data item. For more explanation about **Provenance Queries** and different views they provide on historical data, see [Provenance queries](../../queries/curl/provenance).
+As it mentioned in Orion description [here](../../../introduction) and detailed [here](../../../architecture-and-design/provenance-data), Orion maintains separated persisted graph data structure for historical data transitions, so a user can execute query on those historical changes to understand the lineage of each data item. For more explanation about **Provenance Queries** and different views they provide on historical data, see [Provenance queries](../../queries/curl/provenance).
```

Comment:
As mentioned in the Orion description [here](../../../introduction) and detailed [here](../../../architecture-and-design/provenance-data), Orion maintains a separate persisted graph data structure for historical data transitions, so a user can execute queries on those historical changes to understand the lineage of each data item. For more explanations about **Provenance Queries** and the different views they provide on historical data, see [Provenance queries](../../queries/curl/provenance).